sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5551,7 +5590,8 @@ static pi_result enqueueMemCopyHelper(pi_command_type CommandType,
           pi_cast<std::uintptr_t>(ZeEvent));
   printZeEventList(WaitList);
 
-  if (auto Res = Queue->executeCommandList(CommandList, BlockingWrite))
+  if (auto Res =
+          Queue->executeCommandList(CommandList, BlockingWrite, OkToBatch))
```

Comment:
What are the commands that we still don't batch? Maybe we should just make the last argument (OkToBatch) true by default now?

Reply:
We do have considerable number of call-sites where this argument is 'false' (wait commands, membuffermap, memadvise, etc). So, we don't need to change the default now.